src/highlight/HighlightTarget.tsx

```diff
@@ -61,11 +64,16 @@ const HighlightTarget = (props: Props, ref: React.Ref<HighlightTargetRef>): JSX.
         event.currentTarget.focus();
     };
 
+    React.useEffect(() => {
+        onMount(id);
+    }, []); // eslint-disable-line react-hooks/exhaustive-deps
```

Comment:
We should be really cautious about disabling this rule. If the `id` is regenerated per-render, but is only emitted to the parent on mount, it seems like they would quickly become out of sync. Should the `id` be generated within the effect and stored in state? Or emitted per-render?

Reply:
I don't think this eslint-disable-line directive is needed now. Can we remove it?